OpenLiberty/open-liberty
# 20216 olmvt2

Closes #20216

Test profile:

* openliberty.build.image_fat/fat/src/com/ibm/ws/test/image/suite/FATSuite.java
* openliberty.build.image_fat/fat/src/com/ibm/ws/test/image/suite/InstallServerTest.java
  * validateFeatures
    - Read and display feature information for each server installation.
* openliberty.build.image_fat/fat/src/com/ibm/ws/test/image/suite/ProfilesTest.java
  * testVerifyProfiles
    - Verify that all expected profiles (build.image/profiles) are present.
  * testVerifyProfileImages
    - Verify that each profile has a corresponding installation image.
  * testProfileFeatures
    - Read and display feature information for each profile.

**Still to be done:** Verify that the features of each of the profile specific installations are the full dependency list of features based on the profile features list.
